apps/dashboard/src/components/workflow-editor/steps/email/blocks/digest.tsx

```diff
@@ -11,7 +11,7 @@ export const createDigestBlock = (props: {
 }): BlockItem => {
   const { track, digestStepBeforeCurrent } = props;
 
-  const maxIterations = 5;
+  const maxIterations = 3;
```

Comment:
max iterations for the digest block the same variable is used to subtract from the events count